www/install/php/Update-22.10.0-beta.1.php

```diff
@@ -20,21 +20,27 @@
  */
 
 require_once __DIR__ . '/../../class/centreonLog.class.php';
-
 $centreonLog = new CentreonLog();
 
 //error specific content
 $versionOfTheUpgrade = 'UPGRADE - 22.10.0-beta.1: ';
 $errorMessage = '';
 
 try {
+    $errorMessage = "Impossible to update 'cb_field' table";
```

Comment:
It not linked to this PR, but $errorMessage should be found inside the catch block.
We could log messaes like : start to modify description as info and if the error is thrown log file would contain detailed information about the reason.